freedomofpress/securedrop-https-everywhere-ruleset
# Drop TLD from URL shortcut, add theintercept.com

Closes #2 

Per https://github.com/freedomofpress/securedrop-https-everywhere-ruleset/issues/2#issuecomment-618392602 , updates the naming scheme and adds The Intercept to the ruleset.


### Test plan
1. Open Tor Browser 
2. Type `about:addons` in the URL bar
3. Click "HTTPS Everywhere"
4. Click "Preferences"
5. Click "Update Channels"
6. Type "SecureDrop” and click "Add Update Channel"
7. In JWK section, add release key (`release-pubkey.jwk` )
8. In path prefix, add https://raw.githubusercontent.com/freedomofpress/securedrop-https-everywhere-ruleset/2-update-rules/
9. Click update
- [ ] You see "Stored rulesets version: 2020.4.27" appear
- [ ] `lucyparsonslabs.com.securedrop.tor.onion` and `lucyparsonslabs.securedrop.tor.onion` should redirect you to `http://qn4qfeeslglmwxgb.onion/`
- [ ] `theintercept.securedrop.tor.onion` should redirect you to `http://xpxduj55x2j27l2qytu2tcetykyfxbjbafin3x4i3ywddzphkbrd3jyd.onion/`
- [ ] Other changes introduced here make sense
